### ISSUE TITLE
Initial state of placeholder not right when using ng-model

### ISSUE THREAD

**Issue by serverdevil** [mhartington/ion-md-input#15]
When ng-model is used, the model data and placeholder/label overlap. Once you focus on the input and leave it is all good.
![snip1](https://cloud.githubusercontent.com/assets/4538752/9697649/b9264cd2-53b2-11e5-8256-569477121688.PNG)

**Comment by mhartington** [comment]
Codepen/jsbin/plunker example? 

http://jsbin.com/ximuve/4/edit?html,output

**Comment by serverdevil** [comment]
Well doesn't seem to happen if I edit the jsbin you shared. But can reproduce it when I go on to edit the Demo provided on GitHub. http://codepen.io/serverdevil/pen/avzXbw

**Comment by tomdrewes** [comment]
I've observed that the issue seems related to the timing of the model update.  Here:

http://codepen.io/tomdrewes/pen/xZjJML

If you remove the $timeout, the code works fine.  With the $timeout, the issue occurs.

**Comment by joseluiscampos** [comment]
Im still having this issue with version 1.0.4, I loaded some data from backend then updated my scope variables in order to display its values in a ion-md-item. I cannot remove this "timeout" cause its a backend query for data display.

What I did as a ugly workaround was adding a setTimeout in directive code to wait 1 second before determining if there is a value in the ion-md-item.

 ionic.DomUtil.ready(function() {
         setTimeout(function(){
          if (mdInput.value === '') {
            mdInput.className = mdInput.className.replace(reg, ' ');
          } else {
            mdInput.classList.add(dirtyClass);
          }
         },1000);
        });
